### PR TITLE
Adding driver for Google KMS

### PR DESCRIPTION
**Pull request by funkjedi**
In the [Google Cloud Console](https://console.cloud.google.com/security/kms) create your key ring and key. Make sure your Google IAM user has the `Cloud KMS CryptoKey Encrypter/Decrypter` role for this key.

Copy the Project, Key Ring and Key storing them as `GOOGLE_KMS_PROJECT`, `GOOGLE_KMS_KEY_RING` and `GOOGLE_KMS_KEY` in your local .env file.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "require": {
         "php": "^5.6|^7.0",
         "aws/aws-sdk-php": "^3.0",
-        "illuminate/support": "^5.0|^6.0"
+        "illuminate/support": "^5.0|^6.0",
+        "google/cloud-kms": "^1.7"
```
Comment (jszobody) [inline composer.json:9]: I wonder if we should drop both the aws and google dependencies from here, and instead add instructions in the README alongside each driver. Might be better for users to install only the one they need for the driver they intend to use? Thoughts?

Comment (funkjedi) [reply composer.json:9]: You may be onto something. That thought had occurred to me as well.

Comment (jszobody) [reply composer.json:9]: Yeah, I went ahead and just merged it anyway. It's not a big issue... yet. I'll wait and see how many future drivers are added, split these out in a future release if needed.

**Comment by jszobody** [comment]
@funkjedi Just tagged v1.3, includes this Google KMS driver. Thanks!

**Pull request merged**